src/main/java/com/uid2/client/IdentityMapResponse.java

```diff
@@ -0,0 +1,84 @@
+package com.uid2.client;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.annotations.SerializedName;
+
+import java.util.HashMap;
+
+public class IdentityMapResponse {
+    IdentityMapResponse(String response, IdentityMapInput identityMapInput) {
```

Comment:
nit: Is it worth turning this into a factory function?

Reply:
I don't anticipate implementation differences between UID2 & EUID that wouldn't also require an interface change, and that couldn't be handled server-side.
The one place where we did use factories was Sharing encryption - but even there I think returning whether we're EUID vs UID2 from the server would've been better.

Reply:
Oh, the ctor is package-private. SDK consumers can't create these other than by calling IdentityMapClient.generateIdentityMap or IdentityMapHelper.createIdentityMapResponse